plugins/on-air/README.md

```
@@ -0,0 +1,457 @@
+   - [Step 3 — Add to Your Dashboard](#step-3--add-to-your-dashboard)
+   - [Step 4 — Automate It](#step-4--automate-it)
+6. [Advanced: Custom Labels and Colors](#advanced-custom-labels-and-colors)
+7. [Advanced: Multiple States](#advanced-multiple-states)
+8. [Testing Without Home Assistant](#testing-without-home-assistant)
+9. [Troubleshooting](#troubleshooting)
+
+---
+
+## How It Works
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)